src/styles/ui/CardButton.scss

```diff
@@ -2,11 +2,20 @@
 
 .card {
   &:hover {
-    transform: translateY(-2em);
+    transform: translateY(-1.5em);
   }
   &.whiteCard {
     background-color: $WHITE_CARD_BACKGROUND_COLOR;
     color: $WHITE_CARD_TEXT_COLOR;
+
+    .scrollable-div {
```

Comment:
Why is the whiteCard different from the blackCard?

Reply:
not different, but gets called as child of blackcard.
i do not know how to change. but it is noted

Reply:
was able to just have on of them, perfect